### PR TITLE
Show tooltip if single unit shown in UnitMarkers

### PR DESCRIPTION
**Pull request by Dremars**
- Show tooltip permanently if single unit shown in UnitMarkers
- Show tooltip on unit marker hover

### CODE REVIEW

**Comment by Dremars** [comment]
[Toimipistesivulla kartalla marjan päällä on sama tooltip kuin upotuksessa](https://trello.com/c/Z2PlhssE/334-toimipistesivulla-kartalla-marjan-p%C3%A4%C3%A4ll%C3%A4-on-sama-tooltip-kuin-upotuksessa)

**Comment by Dremars** [comment]
[Kartan marjoille hover tooltip](https://trello.com/c/9E3yfW0m/335-kartan-marjoille-hover-tooltip)

File: src/views/MapView/utils/drawIcon.js [hunk 1]
```diff
@@ -177,7 +177,7 @@ export const drawMarkerIcon = () => {
   const markerIcon = L.icon({
     iconUrl: berryIcon,
     iconSize: [25, 25],
-    iconAnchor: [13, 25],
+    iconAnchor: [13, 33],
```
Comment (mikkoisa) [inline src/views/MapView/utils/drawIcon.js]: Modifying the iconAnchor property makes the icon position less accurate when zooming out. (The icon seems to be moving when zooming out). Adding the tooltipAnchor property is probably better.

**Comment by Dremars** [comment]
Marker cluster feature is going to change our marker rendering #232 rendering this feature obsolete

**Pull request closed without merging**